src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -967,50 +967,12 @@ void grpc_chttp2_mark_stream_writable(grpc_chttp2_transport* t,
   }
 }
 
-static grpc_closure_scheduler* write_scheduler(grpc_chttp2_transport* t,
-                                               bool early_results_scheduled,
-                                               bool partial_write) {
-  // If we're already in a background poller, don't offload this to an executor
-  if (grpc_iomgr_is_any_background_poller_thread()) {
-    return grpc_schedule_on_exec_ctx;
-  }
-  /* if it's not the first write in a batch, always offload to the executor:
-     we'll probably end up queuing against the kernel anyway, so we'll likely
-     get better latency overall if we switch writing work elsewhere and continue
-     with application work above */
-  if (!t->is_first_write_in_batch) {
-    return grpc_core::Executor::Scheduler(grpc_core::ExecutorJobType::SHORT);
-  }
-  /* equivalently, if it's a partial write, we *know* we're going to be taking a
-     thread jump to write it because of the above, may as well do so
-     immediately */
-  if (partial_write) {
-    return grpc_core::Executor::Scheduler(grpc_core::ExecutorJobType::SHORT);
-  }
-  switch (t->opt_target) {
```

Comment:
we can remove this variable from the grpc_chttp2_transport struct now

Reply:
Thanks removed!